src/expiredFruitsReturn.csv

```diff
@@ -0,0 +1,14 @@
+s,banana,100,2020-10-17
+b,banana,10,2020-10-17
+b,banana,10,2020-10-17
+b,banana,10,2020-10-17
+b,banana,10,2020-10-17
+b,banana,10,2020-10-17
+b,banana,10,2020-10-17
+b,banana,10,2020-10-17
+b,banana,10,2020-10-17
+b,banana,10,2020-10-17
+b,banana,10,2020-10-17
+r,banana,50,2020-08-21
+r,banana,50,2020-08-21
+r,banana,50,2020-08-21
```

Comment:
add empty lines to the end of all your files if necessary

Reply:
It's not :)